### PR TITLE
Input type range

### PR DESCRIPTION
**Pull request by feliguez**

### CODE REVIEW

File: src/assets/toolkit/scripts/toolkit.js [hunk 1]
```diff
@@ -30,9 +33,27 @@ $(function() {
     });
   };
 
+  var rangeSlider = function rangeSlider(){
```
Comment (hectorpalmatellez) [inline src/assets/toolkit/scripts/toolkit.js]: There's a space missing after `()` and before `{`

**Comment by hectorpalmatellez** [comment]
I'd rather merge this after #15 so we put this `type="range"` inside `/inputs`

File: src/assets/toolkit/scripts/toolkit.js (hunk 1 above)
Comment (feliguez) [inline src/assets/toolkit/scripts/toolkit.js]: Ok, fix it.

**Comment by hectorpalmatellez** [comment]
:clap:

**Pull request merged**